src/csharp/Microsoft.Spark/Utils/UdfUtils.cs

```diff
@@ -163,6 +163,8 @@ internal static string GetReturnType(Type type)
         internal static JvmObjectReference CreatePythonFunction(IJvmBridge jvm, byte[] command)
         {
             var arrayList = new ArrayList(jvm);
+            var broadcastVariablesList = new ArrayList(jvm);
+            broadcastVariablesList.AddRange(JvmBroadcastRegistry.GetAll());
```

Comment:
If you check the pyspark implementation, it clears the registry once create python function is called. Should we follow the same?

Reply:
I am currently clearing the `JvmBroadcastRegistry` during the constructor call to `UserDefinedFunction`. I am trying to think of a better way/design to do this. Thanks!